docs/training-debug/index.txt

```diff
@@ -1,343 +1,395 @@
+   Confirm that your experiment config does not specify ``resources.slots_per_trial`` or that it is
+   set to ``1``. For example:
 
--  The experiment config is fully validated for cluster-managed experiments, more strictly than it
-   is for ``--local --test`` mode. If you get errors related to ``unmarshaling JSON`` when trying to
-   submit the experiment to the cluster, that is an indication that the experiment config has
-   errors. Please review the :ref:`experiment configuration <experiment-configuration>`.
+   .. code:: yaml
 
-Again, if you are unable to identify the root cause of the issue yourself, please do not hesitate to
-contact Determined through our `community support
+      resources:
+      slots_per_trial: 1
+
+#. Create an experiment with the ``--test`` argument, omitting the ``--local`` argument:
+
+   .. code:: bash
+
+      det experiment create myconfig.yaml my_model_dir --test
+
+#. Perform failure diagnostics:
+
+   If you can run local test mode inside a notebook or shell but are unable to successfully submit
+   an experiment, make sure that notebook or shell customizations you might have made are replicated
+   in your :ref:`experiment configuration <experiment-configuration>`, such as:
+
+   -  If required, a custom Docker image is set in the experiment configuration.
+
+   -  ``pip install`` or ``apt install`` commands needed in the interactive environment are built
+      into a custom Docker image or included in the ``startup-hook.sh`` file in the model definition
+      directory root. See :ref:`startup-hooks` for more information.
+
+   -  Custom bind mounts required in the interactive environment are specified in the experiment
+      configuration.
+
+   -  Environment variables are correctly set in the experiment configuration.
+
+   If no customizations are missing, the following, new layers introduced with a cluster-managed
+   experiment could be the problem cause:
+
+   -  The ``checkpoint_storage`` settings are used for cluster-managed training. If
+      ``checkpoint_storage`` is not configured in the experiment configuration or the master
+      configuration, an error message can occur during experiment configuration validation before
+      the experiment or trials are created. Correct this by providing a ``checkpoint_storage``
+      configuration in one of the following locations:
+
+      -  :ref:`master-configuration`
+      -  :ref:`experiment-configuration`
+
+   -  For a cluster-based experiment, configured ``checkpoint_storage`` settings are validated
+      before training starts. The message ``Checkpoint storage validation failed``, indicates that
+      you should review the ``checkpoint_storage`` setting values.
+
+   -  The experiment configuration is more strictly validated for cluster-managed experiments than
+      for ``--local --test`` mode. Errors related to ``unmarshaling JSON`` when attempting to submit
```

Comment:
Oh!  This error message is no longer what we give to users (thankfully, since it was a terrible message we used to show users).

Instead of ``` ``unmarshalling JSON`` ``` we can say ``` ``invalid experiment configuration`` ``` which is the new pattern that would show up in the command output.

Reply:
Fixed.